bin/create-gxa-analytics-biosolr-lib.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Can you please clarify (and possibly write the clarification in the README) with Alfonso which collections make use of the biosolr-lib? I mention this because maybe the operation should be just generically available instead of being associated by name to `gxa-analytics`.

Reply:
@alfonsomunozpomer I understand this should be generic one for both gxa and scxa ?. I renamed with gxa as it was prefixed with `-create-scxa-anlaytics-*` in `index-scxa` repo.

Reply:
The JAR is stored in Solr’s `.system` collection and needs to be done once. Afterwards it’s available for all collections (the JAR, *NOT* the update processor which needs to be created for each collection). I think this script should be moved somewhere else and `scxa-analytics` and `gxa-analytics` have a symlink instead. IIRC, when you run this script again the JAR blob is overwritten, so it’s good to have the symlink as a dependency, so that we don’t need to create the schemas in a specific order.

Reply:
I agree but removing this from the repo will complicate testing altogether, in the meantime, please @suhaibMo rename this to `add-biosolr-lib.sh` which we can just leave here as a utility for the testing.

Reply:
@pcm32 @alfonsomunozpomer for simplicity, I will have move `create-gxa-analytics-biosolr-lib.sh` to a different name which is more generic.

Reply:
That’s fine; remember to apply the same changes in `index-scxa` (at your earliest convenience, I don’t think it’s really pressing) to keep them in sync. Maybe adding it as a submodule is overengineering.